Aptoide/aptoide-client-v8
# [ASV-327] Showing installed apps with the correct order

**What does this PR do?**

The installed apps on the Apps View were not correctly sorted ( they were Z-A). Now they are correctly sorted (A-Z)

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] AppsAdapter.java

**How should this be manually tested?**

 Open the Apps view and you will be able to see that the installed apps section is correctly sorted (A-Z).

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-327](https://aptoide.atlassian.net/browse/ASV-327)

**Questions:**


**Code Review Checklist**
